hack/generator/pkg/astmodel/function_test.go

```diff
@@ -30,11 +30,11 @@ func (fake *FakeFunction) AsFunc(codeGenerationContext *CodeGenerationContext, r
 }
 
```

Comment:
I think we missed a make all (`go fmt`) in some other PR which is where this came from